src/components/LHNOptionsList/OptionRowLHN.tsx

```diff
@@ -146,53 +107,50 @@ function OptionRowLHN(props) {
-    const fullTitle = isGroupChat ? getGroupChatName(ReportUtils.getReport(optionItem.reportID)) : optionItem.text;
-
-    const subscriptAvatarBorderColor = props.isFocused ? focusedBackgroundColor : theme.sidebar;
+    const isGroupChat = optionItem.type === CONST.REPORT.TYPE.CHAT && optionItem.chatType && !optionItem.isThread && (optionItem.displayNamesWithTooltips?.length ?? 0) > 2;
+    const fullTitle = isGroupChat ? getGroupChatName(isNotEmptyObject(report) ? report : null) : optionItem.text;
 
+    const subscriptAvatarBorderColor = isFocused ? focusedBackgroundColor : theme.sidebar;
     return (
         <OfflineWithFeedback
             pendingAction={optionItem.pendingAction}
-            errors={optionItem.allReportErrors}
+            errors={optionItem.allReportErrors ?? undefined}
             shouldShowErrorMessages={false}
             needsOffscreenAlphaCompositing
         >
             <Hoverable>
                 {(hovered) => (
                     <PressableWithSecondaryInteraction
                         ref={popoverAnchor}
-                        onPress={(e) => {
-                            if (e) {
-                                e.preventDefault();
-                            }
+                        onPress={(event) => {
+                            event?.preventDefault();
                             // Enable Composer to focus on clicking the same chat after opening the context menu.
                             ReportActionComposeFocusManager.focus();
-                            props.onSelectRow(optionItem, popoverAnchor);
+                            onSelectRow(optionItem, popoverAnchor);
                         }}
-                        onMouseDown={(e) => {
+                        onMouseDown={(event) => {
                             // Allow composer blur on right click
-                            if (!e) {
+                            if (!event) {
                                 return;
                             }
 
                             // Prevent composer blur on left click
-                            e.preventDefault();
+                            event.preventDefault();
                         }}
                         testID={optionItem.reportID}
-                        onSecondaryInteraction={(e) => {
-                            showPopover(e);
+                        onSecondaryInteraction={(event) => {
+                            showPopover(event);
                             // Ensure that we blur the composer when opening context menu, so that only one component is focused at a time
                             if (DomUtils.getActiveElement()) {
-                                DomUtils.getActiveElement().blur();
+                                (DomUtils.getActiveElement() as HTMLElement | TextInput)?.blur();
```

Comment:
Is it possible to update `DomUtils.getActiveElement()` return type to escape assertion?

Reply:
I think its not possible (or I dont know how to do it) 
```
const getActiveElement: GetActiveElement = () => document.activeElement;
```
and `activeElement` is returning `Element | null`